CONTRIBUTING.md

```diff
@@ -238,7 +238,47 @@ put us into an unreleasable state
 * Unit tests:
   * Coverage should remain the same or increase
  
-## Pull Request Workflow
+## Running Acceptance Tests
+
+1. Set KUBECONFIG for both minikube and acceptance tests (it will be generated at minikube's start if it does not exist):
+
+```
+export KUBECONFIG=/tmp/minikubeconfig
+```
+
+2. Start minikube:
+
+```
+minikube start
```

Comment:
I think this needs to be changed to `hack/start-minikube.sh`, since running acceptance tests requires the use of a user we set up on cluster creation.

Reply:
Done.
37b19158d3406ba1c82e78df1d199533d78cf09c